packages/aggrid/scss/ix-aggrid.scss

```diff
@@ -60,19 +60,40 @@
         mask-repeat: no-repeat;
         mask-position: center;
         mask-size: contain;
+      }
+      &:hover:not(:disabled)::before {
+        background-color: var(--theme-checkbox-checked--background--hover);
+        border-color: var(--theme-checkbox-checked--border-color--hover);
+      }
+    }
 
-        &:hover:not(:disabled) {
-          background-color: var(--theme-checkbox-checked--background--hover);
-          border-color: var(--theme-checkbox-checked--border-color--hover);
-        }
+    &.ag-indeterminate {
+      position: relative;
+
+      &::before {
+        content: '';
+        position: absolute;
+        inset: 0;
+        background-color: var(--theme-checkbox-checked--background);
+        //TODO: Replace base64 encoded image with svg and css variables
+        //Encoded image: <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg"><rect x="2" y="7" width="12" height="2" fill="#000028"/></svg>
```

Comment:
I think we should remove the comments here. If you want to still have the todo create a jira ticket for that otherwise we will forget to remove this.